<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarsViewModel</class>
 <widget class="QWidget" name="CarsViewModel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Сортировать:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="expensiveFirstRadioButton">
           <property name="text">
            <string>Сначала дороже</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="cheaperFirstRadioButton">
           <property name="text">
            <string>Сначала дешевле</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="morePowerFirstRadioButton">
           <property name="text">
            <string>Сначала мощнее</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="classHigherFirstRadioButton">
           <property name="text">
            <string>По возрастанию класса</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="classLowerFirstRadioButton">
           <property name="text">
            <string>По убыванию класса</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>943</width>
          <height>467</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_2"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,3,3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="showMoreButton">
       <property name="text">
        <string>Показать еще</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
